### PR TITLE
V2 portal db schema updates

### PR DESCRIPTION
**Pull request by commoddity**
Updates PHD to include recently settled-on portal DB updates. These include:

- the chain_alias_domains table and relevant struct updates
- the account_user_access table changes to keep permissions at the portal app level

### CODE REVIEW

File: router/router.go [hunk 1]
```diff
@@ -60,12 +60,14 @@ func NewRouter(config Config) (*Router, error) {
 	rt.Router.HandleFunc("/", rt.HealthCheck).Methods(http.MethodGet)
 	rt.Router.HandleFunc("/reset_cache", rt.ResetCache).Methods(http.MethodPost)
 
+	// TODO update Blockchain create/update endpoints to /v2
+	// rt.Router.HandleFunc("/v1/blockchain", rt.CreateBlockchain).Methods(http.MethodPost)
+	// rt.Router.HandleFunc("/v1/blockchain/{id}", rt.UpdateBlockchain).Methods(http.MethodPut)
+	// rt.Router.HandleFunc("/v1/blockchain/{id}/activate", rt.ActivateBlockchain).Methods(http.MethodPost)
```
Comment (ricantar) [inline router/router.go:66]: Why are they commented? We'll leave it out until v2?

**Comment by ricantar** [comment]
By this point we should call this portal-v2.1/v2.2 😂, thank you adding this update, everything looks good, just the minor comment on the endpoints

File: router/router.go (hunk 1 above)
Comment (commoddity) [reply router/router.go:66]: I will update these in a future separate PR soon (before merging to V2). Sorry should have added comments to explain.

The reasoning is that the only thing that adds/updates Blockchains is the stuff that infra does and it's never via a backend services so not point going through all the trouble of maintaining the legacy adapters for this case. Makes more sense to just create `v2` endpoints for the create/update Chains operations.

Comment (ricantar) [reply router/router.go:66]: Got it, then thsi is ready to go

**Review by ricantar — APPROVED** [review]

**Pull request merged**